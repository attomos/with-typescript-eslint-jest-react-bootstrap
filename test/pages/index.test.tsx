import React from "react";
import { render } from "../testUtils";
import IndexPage from "../../pages/index";

describe("IndexPage", () => {
  it("matches snapshot", () => {
    const { asFragment } = render(<IndexPage />, {});
    expect(asFragment()).toMatchSnapshot();
  });
});
