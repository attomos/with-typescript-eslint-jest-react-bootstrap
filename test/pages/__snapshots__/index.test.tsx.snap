// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`IndexPage matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="container"
  >
    <div
      class="row"
    >
      <header>
        <nav
          class="navbar navbar-expand-lg navbar-light bg-light fixed-top"
        >
          <a
            class="navbar-brand"
            href="#"
          >
            Home
          </a>
        </nav>
      </header>
    </div>
    <div
      class="row"
      style="margin-top: 56px;"
    >
      <main>
        <div
          class="text-gray"
        >
          Next.js TypeScript + ESLint + Jest + react-bootstrap
        </div>
      </main>
    </div>
  </div>
</DocumentFragment>
`;
